### DISCUSSION TITLE
User management

### DISCUSSION THREAD

**Discussion by lucaberti** [gramps-project/gramps-web#483] [Ideas]
Hello, 
Gramps Web is such a great project and all people that are developping it have my gratitude and my respect. It would be nice to have a more refined user management, where admins can easily change emails for other users, force them to change password, change password and (I would say moreover) delete them.
I think this will give admins with less CLI experience the chance to effectivelly manage users (and give advanced users less headhake ;))

**Comment by DavidMStraub** [comment]
Agreed. Some comments:

> where admins can easily change emails for other users

Already possible

> Force them to change password

Why, actually?

> change password 

And then e-mail it to them? I think users should manage their own passwords.

>  delete them

https://github.com/gramps-project/gramps-web/issues/304
It's quite easy to implement but hasn't been high on the priority list.

What's also important is
https://github.com/gramps-project/gramps-web/issues/225
https://github.com/gramps-project/gramps-web/issues/376

**Reply by lucaberti** [reply]
> Already possible

True, sorry!

> > Force them to change password

I.e. when you set up a password for them, but you want them to change with their own.  Or when you know they use "1234" or "password"
This could be combined with a password strength checking system and restrictions.

> > change password
> 
> And then e-mail it to them? I think users should manage their own passwords.
 
Sometimes is easier this way, believe me ;))

> > delete them
> 
> #304 It's quite easy to implement but hasn't been high on the priority list.
> 
> What's also important is #225 #376

Thank you
